packages/web-app-skeleton/src/search/github/client.ts

```diff
@@ -0,0 +1,29 @@
+import { HttpClient } from 'web-pkg'
+
+export interface Repository {
+  id: number
+  name: string
+  full_name: string
+  html_url: string
+  topics: string[]
+}
+
+interface RepositoriesSearch {
+  total_count: number
+  items: Repository[]
+}
+
+export class GitHubClient {
+  private readonly httpClient: HttpClient
+
+  constructor(httpClient: HttpClient) {
```

Comment:
you can use short hand constructor in typescript like this:
`constructor(
    private readonly httpClient: HttpClient
  ){}`

Reply:
Then you don't can remove line 17

Reply:
not a fan of that shorthand, in my opinion wayyyy to implicit.

Reply:
fine for me too :)